var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -419,6 +426,10 @@ def cmake_args(self):
             # default is off; required for bindings.
```

Comment:
Since you added the `requires` above, maybe we can remove the `or spec.satisfies("+bindings")` on the line above?

Reply:
done